<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractorClass</class>
 <widget class="QMainWindow" name="ExtractorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extractor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Delete Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInsert">
        <property name="text">
         <string>Insert Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEdit">
        <property name="text">
         <string>Edit Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Save File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Extractor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonDelete</sender>
   <signal>clicked()</signal>
   <receiver>ExtractorClass</receiver>
   <slot>onDeleteTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>722</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInsert</sender>
   <signal>clicked()</signal>
   <receiver>ExtractorClass</receiver>
   <slot>onInsertTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEdit</sender>
   <signal>clicked()</signal>
   <receiver>ExtractorClass</receiver>
   <slot>onEditTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ExtractorClass</receiver>
   <slot>onSaveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSaveFile()</slot>
 </slots>
</ui>
